src/traces/histogram2dcontour/defaults.js

```diff
@@ -30,5 +31,6 @@ module.exports = function supplyDefaults(traceIn, traceOut, defaultColor, layout
     if(autocontour) coerce('ncontours');
     else coerce('contours.size');
```

Comment:
Can this line get the 🔪 ? It looks like `handleContoursDefaults` re-coerces this, but I could be mistaken.